addons/sale_project/models/sale_order_line.py

```diff
@@ -176,6 +176,8 @@ def _timesheet_create_project(self):
             values['name'] = "%s - %s" % (values['name'], self.product_id.project_template_id.name)
             # The no_create_folder context key is used in documents_project
             project = self.product_id.project_template_id.with_context(no_create_folder=True).copy(values)
+            tasks = self.env['project.task'].with_context(active_test=False).search([('project_id', '=', project.id)])
+            tasks.write({'active': True})
```

Comment:
```suggestion
            project.with_context(active_test=False).tasks.active = True
```
Doesn't that accomplish the same thing?